Docs/details/santactl.md

```diff
@@ -0,0 +1,422 @@
+# santactl
+
+This may be the most complex part of Santa. It really has two types of work that is does. Firstly it contains all of the code and functionality for syncing with a sync-server. The details of which are covered in the [syncing.md](syncing.md) document. This document will cover the status work that santactl performs.
+
+The santactl binary can be used to view the state and configuration of Santa as a whole. It can also inspect individual files. When running without a sync-server it also a supported method of managing the rules database. 
```

Comment:
It does two types of work:

1. It contains all of the code and functionality for syncing with a sync-server.  
2. It can be used to view the state and configuration of Santa as a whole. (...)

The details of santactl's syncing functionality are covered in the syncing.md document.  This document will cover the status work that santactl performs.